install.sh

```diff
@@ -122,6 +125,7 @@ function help() {
 	  ${BOLD}-h, --help${RESET}:         display this help
 
 	${BLUE}${BOLD}Environment variables:${RESET}
+	  ${BOLD}LIQO_REPO${RESET}:          the repository of Liqo to install. Defaults to "liqotech/liqo", but can be change in case of forks.
```

Comment:
```suggestion
	  ${BOLD}LIQO_REPO${RESET}:          the repository of Liqo to install. Defaults to "liqotech/liqo", but can be changed in case of forks.
```